### PR TITLE
Create a Github issue template

### PR DESCRIPTION
**Pull request by nikmacintosh**
Completes #1. Created a barebones issue template, based on the example shown in the docs. We can always tweak this going forward, as our requirements become clearer 🤓

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Expected Behavior
```
Comment (aliyabear) [inline .github/issue_template.md]: Should we add a title to it?

# Issue Description/Title (Emoticon to express our reaction to finding the issue)

Comment (nikmacintosh) [reply .github/issue_template.md]: Ah yes, definitely!

Comment (nikmacintosh) [reply .github/issue_template.md]: Actually, question! When creating an issue, the form fields are essentially Title + Description. Should we repeat those fields in the template?

screen shot 2017-05-06 at 1 25 31 pm

Comment (aliyabear) [reply .github/issue_template.md]: Oh, that's a good idea! However, emoticons must be in there somewhere 

😈

Comment (nikmacintosh) [reply .github/issue_template.md]: Workin' on it ;) I think you'll like the next PR...

**Pull request merged**